configure.ac

```diff
@@ -114,6 +114,9 @@ AC_ARG_ENABLE(static,
 AC_ARG_ENABLE(unix_socket,
   [AS_HELP_STRING([--disable-unix-socket], [Disable unix domain socket])])
 
+AC_ARG_ENABLE(enable_linking_jemalloc,
+  [AS_HELP_STRING([--enable_linking_jemalloc], [Link jemalloc as the memory allocator implementation])])
```

Comment:
help string should be dashes not underscores. Also maybe just --enable-jemalloc ?

Reply:
Done